### PR TITLE
Remove deprecated APIs

### PR DESCRIPTION
**Pull request by bergie**
The purpose of this PR is to make noflo-ui not rely on [deprecated NoFlo APIs](https://github.com/noflo/noflo/blob/master/CHANGES.md) that will be removed in 1.x

### CODE REVIEW

File: graphs/DirectRuntime.fbp [hunk 1]
```diff
@@ -7,8 +7,9 @@ HoldConnection(ui/HoldRuntimeConnection) CONTEXT -> IN ConnectedContext(core/Mer
 'false' -> REQUIREGRAPHS HoldConnection
 
 # Read runtime definition from context and connect
-'runtime' -> KEY GetDefinition(objects/GetObjectKey)
+'runtime' -> STRING SendRuntimeKey(strings/SendString) OUT -> KEY GetDefinition(objects/GetObjectKey)
```
Comment (jonnor) [inline graphs/DirectRuntime.fbp]: What is this SendString? Looks just like `core/Kick`...

File: graphs/ProjectRuntime.fbp [hunk 2]
```diff
@@ -8,8 +8,9 @@ HoldConnection(ui/HoldRuntimeConnection) CONTEXT -> IN ConnectedContext(core/Mer
 'true' -> REQUIREGRAPHS HoldConnection
 
 # If not, the connect
-'runtime' -> KEY GetDefinition(objects/GetObjectKey)
+'runtime' -> STRING SendRuntimeKey(strings/SendString) OUT -> KEY GetDefinition(objects/GetObjectKey)
```
Comment (jonnor) [inline graphs/ProjectRuntime.fbp]: Why do we need this now, and not before?

File: graphs/DirectRuntime.fbp (hunk 1 above)
Comment (bergie) [reply graphs/DirectRuntime.fbp]: Yes, very similar, but with two differences:

* SendString is typed, so we know what we're sending is a string
* SendString sends on data packet, Kick sends on complete stream

File: graphs/ProjectRuntime.fbp (hunk 2 above)
Comment (bergie) [reply graphs/ProjectRuntime.fbp]: The original, pre Process API noflo-objects treated the `KEY` port as essentially a control port. The new one treats it as regular input. This is great for getting rid of race conditions, but the downside is that graphs using the component(s) have to be adapted like we did here

Comment (jonnor) [reply graphs/ProjectRuntime.fbp]: Ok

File: graphs/DirectRuntime.fbp (hunk 1 above)
Comment (jonnor) [reply graphs/DirectRuntime.fbp]: Ok, thanks for explaining

**Pull request merged**